src/status_im/ui/screens/profile/events.cljs

```diff
@@ -169,17 +85,23 @@
 (handlers/register-handler-fx
   :my-profile/save-profile
   (fn [{:keys [db now]} _]
-    (let [{:accounts/keys [accounts current-account-id]} db
-          {old-status :status} (get accounts current-account-id)
-          {:keys [status photo-path]} (:my-profile/profile db)
+    (let [{:keys [photo-path]} (:my-profile/profile db)
           cleaned-name (clean-name db :my-profile/profile)
           cleaned-edit (merge {:name         cleaned-name
-                               :status       status
                                :last-updated now}
                               (if photo-path
                                 {:photo-path photo-path}))]
       (-> (clear-profile {:db db})
           (accounts-events/account-update cleaned-edit)
-          (status-change {:old-status old-status
-                          :status     status})
           (update :dispatch-n concat [[:navigate-back]])))))
+
+(handlers/register-handler-fx
+  :group-chat-profile/start-editing
+  (fn [{:keys [db]} _]
+    {:db (assoc db :group-chat-profile/editing? true)}))
```

Comment:
Minor point, but both `:group-chat-profile/start-editing` and `:group-chat-profile/save-profile` events could be defined via `handlers/register-handler-db` which would make them little bit less verbose.